### DISCUSSION TITLE
Documentation in English

### DISCUSSION THREAD

**Discussion by TorstenDittmann** [swoole/swoole-src#4502] [Ideas]
At work I use Swoole in our open source project and thus depend on the documentation.

Now with the current `swoole`/`openswoole` situation, the English documentation under the co.uk domain comes with several misconceptions regarding the official repository.

Since complaining won't help the situation, here comes my suggestion. As our team and me is experienced with Swoole, I would translate the official Chinese documentation into English and release it as a public repo together with a doc-generator.

This is needed for our team as a knowledge base in the first place, but if there is interest we can gladly keep the English documentation up to date together.

**Comment by matyhtf** [comment]
We are working on a plan to open source Chinese documents. And start translating English documents. All this will be processing on github. We are going to use swoole.dev as the new official website of the open source project.

After the preparation work is completed, You can submit a pull request to the github repository.

**Reply by TorstenDittmann** [reply]
That's really amazing to hear, feel free to reach out if any help is needed 🙌🏻
